tools/api/generate_go_protobuf.py

```diff
@@ -142,3 +140,5 @@ def updated(repo):
             new_sha = changes[0]
             write_revision_info(repo, new_sha)
             publish_go_protobufs(repo, new_sha)
+    else:
```

Comment:
nit: bail early - ie `if not args.sync: print + return`

then you can dedent the following code

Reply:
actually sys.exit i guess

Reply:
Good suggestion, extra commit pushed with those changes.